schema.graphql

```diff
@@ -1,27 +1,32 @@
 type Contribution @entity {
   id: ID!
-
+  extrinsicHash: String! @index
   paraId: Int! @index
-  vaultId: String!
+  vault: Vaults! @index
   blockHeight: Int! @index
   account: String! @index
   amount: String!
-  referralCode: String
   timestamp: Date!
 }
 
 type Vaults @entity {
```

Comment:
can we add firstSlot and lastSlot field in Vault schema?

Reply:
But the `id` includes that info.

Reply:
I think it's a different meaning actually even we use it as vault id

Reply:
Okay, got it. Should prompt the staging to production first?